install/windows.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_label: Windows
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Installing RGBDS on Windows
+
+The install instructions are a bit different depending on the environment in which you wish to use RGBDS.
+
+<Tabs>
+<TabItem value="wsl" label="WSL">
+
+Please refer to [the install instructions](linux.md) for your WSL's running Linux distribution.
+You can check what your WSL distribution is by running `wsl -l -v` in the Command Prompt.
+(The default Linux distribution on WSL is Ubuntu, whose package manager is `apt`.)
+
+</TabItem>
+<TabItem value="cygwin" label="Cygwin / MSYS2">
+
+1. Go the [latest release](https://github.com/gbdev/rgbds/releases/latest) (or pick a specific version from [the list](https://github.com/gbdev/rgbds/releases)).
+2. Under "Assets" at the bottom, download either <code>rgbds-<var>&lt;version&gt;</var>-win32.zip</code> (for 32-bit Windows) or <code>rgbds-<var>&lt;version&gt;</var>-win64.zip</code> (for 64-bit Windows).
+   (For example, version 0.7.0 for 64-bit Windows would have `rgbds-0.7.0-win64.zip`).
+3. Unzip the .zip file.
+   It should give you the RGBDS `.exe` files and a couple of `.dll` files.
+4. Copy all of those `.exe` and `.dll` files to the `/usr/local/bin` directory of your Cygwin/MSYS2 installation.
+   (You can learn its equivalent Windows path by running `cygpath -w /usr/local/bin` in the Cygwin terminal.)
+
+   :::caution
+
+   Do not put them in a subdirectory (e.g. `/usr/local/bin/rgbds/`)!
+   This would not work.
+
+   :::
+
+After that, you should be able to use RGBDS from within the Cygwin/MSYS2 terminal, which you can confirm by running `rgbasm -V`.
+
+If `rgbasm -V` doesn't work, check that `/usr/local/bin` is listed in your Cygwin/MSYS2 `PATH` (run `echo $PATH` to check).
+If it isn't listed, you must add it (e.g. run `echo 'export PATH="/usr/local/bin:$PATH"' >> ~/.bashrc`, then close your Cygwin terminal and open a new one).
+
+:::tip
+
+If you can choose between using Cygwin or MSYS2, be advised that Cygwin is slower and has been reported to cause a bit of trouble to some.
+
+:::
+
+</TabItem>
+<TabItem value="win32" label="None of those">
+
```

Comment:
```suggestion

Using just the pre-built Windows executables is recommended if all you need is RGBDS' four programs.

```

Reply:
Wouldn't be this the whole idea of those install instructions (or at least the "default"?)